webview/src/plots/components/util.ts

```diff
@@ -43,3 +48,22 @@ export const changeDragAndDropMode = (
       : toggleCustomDragAndDrop
   return dispatch(toggleMode(!isDragAndDropMode))
 }
+
+const cleanTitlePart = (title: string) => {
```

Comment:
There is a bit of duplication between this and `splitColumnPath`, but it's not possible to import this function inside the webview.